superset-frontend/src/components/AvatarIcon.tsx

```diff
@@ -19,15 +19,16 @@
 import React from 'react';
 import styled from '@superset-ui/style';
 import { getCategoricalSchemeRegistry } from '@superset-ui/color';
-import { Tooltip, OverlayTrigger } from 'react-bootstrap';
 import Avatar, { ConfigProvider } from 'react-avatar';
+import TooltipWrapper from 'src/components/TooltipWrapper';
```

Comment:
Just a comment that's out of scope for this PR, and a bit pedantic, but it might be nice to just call this component `Tooltip` since I suspect _most_ of our Superset components going forward will be wrappers for some react-bootstrap/antd/whateverLibrary component. I would just name the tool semantically, lest we start calling everything a Wrapper.

Reply:
... and if this was due to namespacing issues, I would just rename the internal import, e.g. `import { Tooltip as RB_Tooltip } from 'react-bootstrap';

Reply:
The reason it's tooltipWrapper is because it just allows for any `tooltip` to be rendered with the correct triggers. `tooltip` is a prop of tooltipWrapper that defaults to using the bootstrap tooltip.